src/apps/answers/filters.py

```diff
@@ -34,6 +34,7 @@ class AppletSubmissionsFilter(BaseQueryParams):
 class AppletSubmitDateFilter(BaseQueryParams):
     respondent_id: uuid.UUID | None
     target_subject_id: uuid.UUID | None
+    activity_or_flow_id: str | None
```

Comment:
Shouldn't this remain `uuid.UUID` to maintain validation rules? I don't see anywhere else in the codebase where we type `activity_or_flow_id` as `str`.

Were you running into a typing issue that prompted you to switch to `str`?